### DISCUSSION TITLE
InvalidDNSError:

### DISCUSSION THREAD

**Discussion by kyoyacchi** [community/community#57436] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

My website was working fine but recently it stopped working after some time. 

Github says "Both kyoya.ga and its alternate name are improperly configured
Domain's DNS record could not be retrieved. For more information, see documentation (InvalidDNSError)."
It also says "Certificate already exists for kyoya.ga and is usable. " When i click on" check again."
![Screenshot_20230607-174203](https://github.com/community/community/assets/63583961/8ff64d41-c432-4da5-9872-5aef67121347)

I'm using Freenom for my website. 
These are my dns configurations.
https://kyoya.ga/
![Screenshot_20230607-173435_1_1m_1](https://github.com/community/community/assets/63583961/d92e4b18-b30d-4842-b336-6fa6e3f33b17)

![Screenshot_20230607-174243](https://github.com/community/community/assets/63583961/5a108d7a-6cd9-4789-90ca-8235e1fc7ae7)

**Comment by keehl1213** [comment]
Same problem. Have you resolved?

**Reply by kyoyacchi** [reply]
No, i haven't..

**Reply by keehl1213** [reply]
Found the answer.  Freenom was sued by meta.  Check this [article](https://www.netcraft.com/blog/impact-of-freenom-halting-registrations-on-cybercrime/).
